#request
active_slam/AvailablePaths paths
int32 fov_angle_deg
int32 kind_cost
bool only_last
---
#response
active_slam/SinglePath path
float64 entropy_path_
active_slam/PathInfo[] grouped_paths 
active_slam/PathInfo best_path